src/xstate/mod.rs

```diff
@@ -686,7 +686,10 @@ impl XState {
         motif_hints: Option<motif::Hints>,
         has_transient_for: bool,
     ) -> XResult<bool> {
+        let mut no_function_motif = false;
         if let Some(hints) = motif_hints {
+            // indicating motif flags first bit is not set (eg. 0x2 decorations only)
+            no_function_motif = hints.functions.is_none();
```

Comment:
```suggestion
            no_function_motif = hints.functions.is_none_or(|f| f.is_empty());
```
In the name of future proofing.